src/Microsoft.Extensions.Localization/ResourceManagerWithCultureStringLocalizer.cs

```diff
@@ -6,11 +6,12 @@
 using System.Globalization;
 using System.Reflection;
 using System.Resources;
+using Microsoft.Extensions.Localization.Internal;
 
 namespace Microsoft.Extensions.Localization
 {
     /// <summary>
-    /// An <see cref="IStringLocalizer"/> that uses the <see cref="System.Resources.ResourceManager"/> and
+    /// An <see cref="IStringLocalizer"/> that uses the <see cref="ResourceManager"/> and
```

Comment:
When non-design, do this fix for `ResourceReader` and in `ResourceManagerStringLocalizer`.